services/director-v2/src/simcore_service_director_v2/models/domains/dynamic_sidecar.py

```diff
@@ -0,0 +1,78 @@
+from typing import Dict, Any, Optional, List
+
+from pydantic import BaseModel, Field, validator
+
+ComposeSpecModel = Optional[Dict[str, Any]]
+
+
+class PathsMappingModel(BaseModel):
+    inputs_path: str = Field(
+        ..., description="path where the service expects all the inputs folder"
+    )
+    outputs_path: str = Field(
+        ..., description="path where the service expects all the outputs folder"
+    )
+    other_paths: List[str] = Field(
+        [],
+        description="optional list of path which contents need to be saved and restored",
+    )
+
+    @validator("other_paths", always=True)
+    @classmethod
+    def other_paths_always_a_list(cls, v):
+        return [] if v is None else v
+
+
+class StartDynamicSidecarModel(BaseModel):
+    user_id: str
+    project_id: str
+    service_key: str
+    service_tag: str
```

Comment:
for these as well:
see here packages/models-library/src/models_library/projects_nodes.py